src/data/ACTIONS/DNC.js

```diff
@@ -1,3 +1,236 @@
 export default {
-	// TODO
+
+	SINGLE_STANDARD_FINISH: {
+		id: 16191,
+		icon: 'https://xivapi.com/i/003000/003459.png',
+		name: 'Single Standard Finish',
+		onGcd: true,
+	},
+	DOUBLE_STANDARD_FINISH: {
+		id: 16192,
+		icon: 'https://xivapi.com/i/003000/003459.png',
+		name: 'Double Standard Finish',
+		onGcd: true,
+	},
+	SINGLE_TECHNICAL_FINISH: {
+		id: 16193,
+		icon: 'https://xivapi.com/i/003000/003474.png',
+		name: 'Single Technical Finish',
+		onGcd: true,
+	},
+	DOUBLE_TECHNICAL_FINISH: {
+		id: 16194,
+		icon: 'https://xivapi.com/i/003000/003474.png',
+		name: 'Double Technical Finish',
+		onGcd: true,
+	},
+	TRIPLE_TECHNICAL_FINISH: {
+		id: 16195,
+		icon: 'https://xivapi.com/i/003000/003474.png',
+		name: 'Triple Technical Finish',
+		onGcd: true,
+	},
+	QUADRUPLE_TECHNICAL_FINISH: {
+		id: 16196,
+		icon: 'https://xivapi.com/i/003000/003474.png',
+		name: 'Quadruple Technical Finish',
+		onGcd: true,
+	},
+	CASCADE: {
+		id: 15989,
+		icon: 'https://xivapi.com/i/003000/003451.png',
+		name: 'Cascade',
+		onGcd: true,
+		combo: {
+			start: true,
+		},
+	},
+	FOUNTAIN: {
+		id: 15990,
+		icon: 'https://xivapi.com/i/003000/003452.png',
+		name: 'Fountain',
+		onGcd: true,
+		combo: {
+			from: 15989,
+			end: true,
+		},
+	},
+	REVERSE_CASCADE: {
+		id: 15991,
+		icon: 'https://xivapi.com/i/003000/003460.png',
+		name: 'Reverse Cascade',
+		onGcd: true,
+	},
+	FOUNTAINFALL: {
+		id: 15992,
+		icon: 'https://xivapi.com/i/003000/003464.png',
+		name: 'Fountainfall',
+		onGcd: true,
+	},
+	WINDMILL: {
+		id: 15993,
+		icon: 'https://xivapi.com/i/003000/003453.png',
+		name: 'Windmill',
+		onGcd: true,
+		combo: {
+			start: true,
+		},
+	},
+	BLADESHOWER: {
+		id: 15994,
+		icon: 'https://xivapi.com/i/003000/003461.png',
+		name: 'Bladeshower',
+		onGcd: true,
+		combo: {
+			from: 15993,
+			end: true,
+		},
+	},
+	RISING_WINDMILL: {
+		id: 15995,
+		icon: 'https://xivapi.com/i/003000/003463.png',
+		name: 'Rising Windmill',
+		onGcd: true,
+	},
+	BLOODSHOWER: {
+		id: 15996,
+		icon: 'https://xivapi.com/i/003000/003465.png',
+		name: 'Bloodshower',
+		onGcd: true,
+	},
+	STANDARD_STEP: {
+		id: 15997,
+		icon: 'https://xivapi.com/i/003000/003454.png',
+		name: 'Standard Step',
+		onGcd: false, // it is but it isn't...
+		cooldown: 30,
+	},
+	TECHNICAL_STEP: {
+		id: 15998,
+		icon: 'https://xivapi.com/i/003000/003473.png',
+		name: 'Technical Step',
+		onGcd: false, // it is but it isn't...
```

Comment:
Ah, I follow what you were talking about on discord now. My bad.

When I get some time, I'll check the game data to see how these are represented, it's likely something we can take into account.